kalaponents.variable.inc

```diff
@@ -26,7 +26,7 @@ function kalaponents_variable_info($options = array()) {
     'kalaponents composite' => 'kalaponents_icons',
     'type' => 'multiple',
     'multiple callback' => 'kalaponents_get_types',
-    'title' => t('Kalamuna Component Icons', array(), $options),
+    'title' => t('Icons', array(), $options),
```

Comment:
Perfect from what we said earlier. But actually now I remember why I didn't do it this way from the start. It's because this is a generic Variables module info declaration, and if you look at the variables from the main Variables module config page, semantically speaking, they need to be prefixed with "Kalaponents." Do you mind making that change? Thanks!